client/login/wp-login/index.jsx

```diff
@@ -36,7 +36,7 @@ import RequestLoginEmailForm from '../magic-login/request-login-email-form';
 import { recordTracksEvent } from 'state/analytics/actions';
 import PageViewTracker from 'lib/analytics/page-view-tracker';
 
-class Login extends React.Component {
+export class Login extends React.Component {
```

Comment:
Nop, it's not a problem here, just a comment that it can, sometimes, have side effects ;)